irohad/model/commands/transfer_asset.hpp

```diff
@@ -45,7 +46,7 @@ namespace iroha {
       /**
        * Amount of transferred asset
        */
-      std::string amount;
+      std::decimal::decimal64 amount;
```

Comment:
`clang` doesn't support. maybe ok for now.
but in the future, how is importing `boost::multiprecision`?